### PR TITLE
New version: CompLabs.GoogleDocs version 2022.12.1

### LINKED ISSUE #200041: [Update Request]: CompLabs.GoogleDocs
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

CompLabs.GoogleDocs

### Package Version

v2022.12.1

### Please describe the changes you would like to see

https://github.com/Comp-Labs/Google-Docs/releases

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #200041

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

Image

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/200068)

### CODE REVIEW

File: manifests/c/CompLabs/GoogleDocs/2022.12.1/CompLabs.GoogleDocs.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: CompLabs.GoogleDocs
+PackageVersion: 2022.12.1
+PackageLocale: en-US
+Publisher: Comp Labs
+PublisherUrl: https://complabs.in/
```
Comment (Dragon1573) [inline manifests/c/CompLabs/GoogleDocs/2022.12.1/CompLabs.GoogleDocs.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://techfiddle.io/
```

File: manifests/c/CompLabs/GoogleDocs/2022.12.1/CompLabs.GoogleDocs.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+PublisherSupportUrl: https://github.com/Comp-Labs/Google-Docs/issues
+PrivacyUrl: https://complabs.in/legal/privacy-policy
```
Comment (Dragon1573) [inline manifests/c/CompLabs/GoogleDocs/2022.12.1/CompLabs.GoogleDocs.locale.en-US.yaml:10]:
```suggestion
PrivacyUrl: https://techfiddle.io/legal/privacy-policy
```

File: manifests/c/CompLabs/GoogleDocs/2022.12.1/CompLabs.GoogleDocs.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+PackageName: Unofficial Google Docs
+PackageUrl: https://complabs.in/apps/google-docs
```
Comment (Dragon1573) [inline manifests/c/CompLabs/GoogleDocs/2022.12.1/CompLabs.GoogleDocs.locale.en-US.yaml:12]:
```suggestion
PackageUrl: https://github.com/Comp-Labs/Google-Docs
```

[Project URL](https://techfiddle.io/apps/google-docs) on their [GitHub Repo](https://github.com/Comp-Labs/Google-Docs) is also `HTTP 404 Not Found`.

File: manifests/c/CompLabs/GoogleDocs/2022.12.1/CompLabs.GoogleDocs.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+License: MIT
+LicenseUrl: https://github.com/Comp-Labs/Google-Docs/blob/main/LICENSE.md
```
Comment (Dragon1573) [inline manifests/c/CompLabs/GoogleDocs/2022.12.1/CompLabs.GoogleDocs.locale.en-US.yaml:14]:
```suggestion
LicenseUrl: https://github.com/Comp-Labs/Google-Docs/blob/main/LICENSE
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**